lib/serial_terminal.pm

```diff
@@ -105,7 +105,7 @@ sub select_virtio_console {
     }
 
     bmwqemu::fctwarn("Cannot use virtio console on s390 on ZKVM") if $is_virtio;
-    select_console('root-console');
+    select_console('root-console', ensure_tty_selected => 1);
```

Comment:
Shouldn't be better to split it into separate commit/pr? It is not mentioned in the description and  I don't see relation to ganglia test.